pkg/unikontainers/config.go

```diff
@@ -29,15 +29,15 @@ import (
 
 var ErrEmptyAnnotations = errors.New("spec annotations are empty")
 
-// A config struct that holds the necessary info on how to execute our unikernel
+// A UnikernelConfig struct holds the info provided by bima image on how to execute our unikernel
 type UnikernelConfig struct {
 	UnikernelType   string `json:"com.urunc.unikernel.unikernelType"`
 	UnikernelCmd    string `json:"com.urunc.unikernel.cmdline,omitempty"`
 	UnikernelBinary string `json:"com.urunc.unikernel.binary"`
 	Hypervisor      string `json:"com.urunc.unikernel.hypervisor"`
 }
 
-// GetUnikernelConfig tries to get the Unikernel config from the bundle annotation.
+// GetUnikernelConfig tries to get the Unikernel config from the bundle annotations.
```

Comment:
add a comment about the annotations and also point to the relevant issue (if we don't have one, we should create one)

Reply:
Done